src/assets/main.css

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Source Sans Pro', sans-serif;
+  font-weight: 400;
+}
+img {
+  max-width: 100%;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+  font-family: 'Playfair Display', serif;
+  font-weight: 700;
+}
+.container {
+  width: 95%;
+  max-width: 1180px;
+  margin: 0 auto;
+}
+footer {
+  text-align: left;
+  width: 100%;
+  background-color: #f4f4f4;
+  padding: 15px 2.5%;
+  margin-top: 15px;
+  font-size: 12px;
+}
+.grid {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: auto;
+  grid-gap: 15px 15px;
+}
+@media (min-width: 450px) {
+  .grid-4 {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: auto;
+    grid-gap: 25px 25px;
+  }
+}
+@media (min-width: 550px) {
+  .grid-3 {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: auto;
+    grid-gap: 25px 25px;
+  }
+}
+@media (min-width: 768px) {
+  .grid-2 {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: auto;
+    grid-gap: 25px 25px;
+  }
+  .grid-4 {
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: auto;
+    grid-gap: 25px 25px;
+  }
+}
+header {
+  width: 100%;
+  background-color: #708090;
+  color: #fff;
+  padding: 15px 2.5%;
+}
+.home {
+  text-align: center;
+}
+.categories h1 {
+  margin-top: 40px;
+}
+.cat-group {
+  margin-top: 40px;
+}
+.cat-group h4 {
+  margin-bottom: 20px;
+}
+.cat-group .cat-item {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 10% 15%;
+  background-color: #f4f4f4;
```

Comment:
this color is being used 4 times across this PR, should we make this a variable?

Reply:
I don't see this color being used more than once, can you be more specific please?